lib/deleteForm.js

```diff
@@ -83,4 +86,60 @@ export async function deleteForm(serviceId, sessionUri) {
   `;
   await update(insertTombstoneQuery);
 
+  if (hadConcept && !(await conceptHasInstances(conceptUri))) {
+    await removeInstantiatedFlag(conceptUri);
+  }
+}
+
+async function conceptUriForService(serviceUri) {
+  return (await query(`
+      ${PREFIXES}
+
+      SELECT DISTINCT ?concept
+      WHERE {
+        BIND( ${sparqlEscapeUri(serviceUri)} as ?service)
+        ?service a cpsv:PublicService;
+          dct:source ?concept.
```

Comment:
Minor nitpicks:
* There's an extra space after `BIND(`
* There's no space between the triple objects and `.`/`;`